### PR TITLE
Murano Get-started

### PR DESCRIPTION
**Pull request by chadandycha**
Doc has gone through final review and is ready to go live.

### CODE REVIEW

File: murano/get-started/README.md [hunk 1]
```diff
@@ -4,52 +4,62 @@ template: default
...
+
+* Remotely controllable 
+
+<table>
+  <tr>
+    <td>Murano Products - Getting-started Example Templates</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>![Python Simulator](assets/sim.png)</td>
```
Comment (azdle) [inline murano/get-started/README.md]: We should either re-size the images in this table or keep the style from the previous version. They're quite big:

http://i.imgur.com/GNEbT7T.png

File: murano/get-started/README.md [hunk 2]
```diff
@@ -4,52 +4,62 @@ template: default
...
- * Ability for users to claim ownership of devices
- * Users can see device sensor data and turn the light on/off
- * Users can invite a shared user to access the light.  
+<table>
+  <tr>
+    <td>Murano Solution - Getting-started Example Templates</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>![Consumer Example App](assets/lightapp.png)</td>
```
Comment (azdle) [inline murano/get-started/README.md]: Same here.

File: murano/get-started/README.md [hunk 3]
```diff
@@ -4,52 +4,62 @@ template: default
...
+
+* Connected to the Murano platform
+
+* Reporting sensor data
+
+* Remotely controllable 
+
+<table>
+  <tr>
+    <td>Murano Products - Getting-started Example Templates</td>
```
Comment (azdle) [inline murano/get-started/README.md]: This should get the colspan attribute from the previous version.

File: murano/get-started/README.md [hunk 4]
```diff
@@ -4,52 +4,62 @@ template: default
...
+* Ability for users to invite a shared user to access the lightbulb
 
-The example consumer application provides all the source code, template files, and configuration for building and deploying a connected product and application.  At the end of this walk-through, you should have a deployed web application that has:
- * Unique user sign-up and log-in
- * Ability for users to claim ownership of devices
- * Users can see device sensor data and turn the light on/off
- * Users can invite a shared user to access the light.  
+<table>
+  <tr>
+    <td>Murano Solution - Getting-started Example Templates</td>
```
Comment (azdle) [inline murano/get-started/README.md]: 'colspan' again

**Comment by azdle** [comment]
Really, in general we should probably copy over all the HTML markup from the previous version. The tables look a bit wacky as they are now.

**Comment by dweaver** [comment]
Is this pull request still relevant, @chadandycha? Or should it be closed?

**Comment by chadandycha** [comment]
This pull request is still relevant. The fixes to the tables were made a while back, so this should be ready to be merged. Thanks.

**Pull request merged**